components/AdminNavbar.tsx

```diff
@@ -0,0 +1,181 @@
+'use client';
+import {
+    Box,
+    Button,
+    Flex,
+    IconButton,
+    Stack,
+    Text,
+    Link as ChakraLink,
+    useBreakpointValue,
+} from '@chakra-ui/react';
+import {
+    BarChart,
+    Calendar,
+    LogOut,
+    Menu,
+    Scissors,
+    Users,
+} from 'lucide-react';
+import NextLink from 'next/link';
+import { usePathname } from 'next/navigation';
+import React, { useState } from 'react';
+
+const navItems = [
+    { name: 'Citas', href: '/admin/citas', icon: Calendar },
+    { name: 'Empleados', href: '/admin/empleados', icon: Users },
+    { name: 'Tratamientos', href: '/admin/tratamientos', icon: Scissors },
+    { name: 'Reportes', href: '/admin/reportes', icon: BarChart },
+];
+
+export default function AdminNavbar() {
+    const pathname = usePathname();
+    const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
+    const isDesktop = useBreakpointValue({ base: false, md: true });
+
+    return (
+        <Box bg='gray.800' color='white' shadow='lg'>
+            <Flex
+                maxW='7xl'
+                mx='auto'
+                px={{ base: 4, md: 6, lg: 8 }}
+                h='16'
+                align='center'
+                justify='space-between'
+            >
+                {/* Logo y links */}
+                <Flex align='center'>
+                    <ChakraLink
+                        as={NextLink}
+                        href='/admin/citas'
+                        display='flex'
+                        alignItems='center'
+                        _hover={{ textDecoration: 'none' }}
+                        color='white'
+                    >
+                        <Scissors size={32} />
+                        <Text ml='2' fontSize='xl' fontWeight='bold'>
+                            Elegangster Admin
+                        </Text>
+                    </ChakraLink>
+
+                    {isDesktop && (
```

Comment:
💅🏽 I think you can create two separated components for this, one that contains the menu when is desktop and the other one for when it isn't